### PR TITLE
[BS-5] Add budget widget to home screen 

### PR DESCRIPTION
**Pull request by Valter4578**
- Changed Appearance.swift to make font compatible with SwiftUI and added missing colours 
- Changed project structure: put Home in Modules folder, renamed BalanceFlowViewContainer to HomeContainerView 
- Add budget widget view to home

### CODE REVIEW

File: bonsai/Modules/Home/View/HomeContainerView.swift [hunk 1]
```diff
@@ -9,11 +9,11 @@ import Foundation
 import SwiftUI
 
 
-struct BalanceFlowViewContainer: View {
+struct HomeContainerView: View {
     var body: some View {
         ZStack {
```
Comment (antonhoang) [inline bonsai/Modules/Home/View/HomeContainerView.swift:14]: I think we can add vertical scroll view there

Comment (antonhoang) [inline bonsai/Modules/Home/View/HomeContainerView.swift:14]: I think we can add vertical scroll view there

File: bonsai.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -189,14 +221,15 @@
...
+				DEVELOPMENT_TEAM = 44ZWCLCJ2W;
 				GENERATE_INFOPLIST_FILE = YES;
 				IPHONEOS_DEPLOYMENT_TARGET = 14.0;
 				LD_RUNPATH_SEARCH_PATHS = (
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
 				MARKETING_VERSION = 0.1;
-				PRODUCT_BUNDLE_IDENTIFIER = com.bonsai.bonsai;
+				PRODUCT_BUNDLE_IDENTIFIER = com.bonsai.mrvalter566;
```
Comment (antonhoang) [inline bonsai.xcodeproj/project.pbxproj]: maybe change bundle for organization 
com.app.bonsai or com.appbonsai.bonsai

File: bonsai/Modules/Home/View/Budget/BudgetView.swift [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+//
+//  Created by Максим Алексеев  on 07.12.2021.
+//
+
+import SwiftUI
+
+struct BudgetView: View {
+    var body: some View {
+        VStack(alignment: .leading) {
+            HStack {
```
Comment (antonhoang) [inline bonsai/Modules/Home/View/Budget/BudgetView.swift]: I think its might be better make HStack in separate view to reduce this long line code BudgetView

**Review by antonhoang — COMMENTED** [review]
Also check please font seems like doesn't match with design

**Review by VladimirBrejcha — APPROVED** [review]

**Pull request merged**